lte/gateway/docker/services/python/Dockerfile

```diff
@@ -121,28 +146,30 @@ RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/
 RUN wget -qO - https://packages.fluentbit.io/fluentbit.key | apt-key add -
 
 RUN apt-get update && apt-get install -y \
-  td-agent-bit \
+  bcc-tools \
   libopenvswitch \
   openvswitch-datapath-dkms \
   openvswitch-common \
   openvswitch-switch \
-  bcc-tools \
+  td-agent-bit \
   wireguard \
   && rm -rf /var/lib/apt/lists/*
 
+COPY ./orc8r/gateway/python/scripts/ /usr/local/bin
+COPY ./lte/gateway/python/scripts/ /usr/local/bin
+COPY ./lte/gateway/configs/templates /etc/magma/templates/
+COPY ./orc8r/gateway/configs/templates/nghttpx.conf.template /etc/magma/templates/nghttpx.conf.template
+COPY ./lte/gateway/deploy/roles/magma/files/set_irq_affinity /usr/local/bin/set_irq_affinity
+
 COPY --from=builder /build /build
-COPY --from=builder /magma /magma
-COPY --from=builder /magma/orc8r/gateway/python/scripts/ /usr/local/bin
-COPY --from=builder /magma/lte/gateway/python/scripts/ /usr/local/bin
+COPY --from=builder /magma/lte/gateway/python/magma /magma/lte/gateway/python/magma
+COPY --from=builder /magma/orc8r/gateway/python/magma /magma/orc8r/gateway/python/magma
 COPY --from=builder /var/tmp/python3-aioeventlet* /var/tmp/
```

Comment:
we probably want to make it clearer that we copy over some debs here.

Ike copying /var/tmp/python3-aioeventlet*.deb